src/wwwroot/partials/settings/my-plan.html

```diff
@@ -41,7 +41,8 @@
       </div>
       <div class="flex two flex--wrap">
         <label class="width--full special uppercase">{{ 'my_plan_renewal_date' | translate }}</label>
-        <p class="renewal-date">{{ vm.resetDate | date :'mediumDate' :'+0' }}</p>
+        <p class="renewal-date" ng-if="!vm.isAccountClosed">{{ vm.resetDate | date :'mediumDate' :'+0' }}</p>
+        <p class="renewal-date" ng-if="vm.isAccountClosed">{{ 'my_plan_account_closed' | translate }}</p>
```

Comment:
I think that it should always show a date (see [previous comment](https://github.com/DopplerRelay/relay-webapp/pull/111/files#diff-5f616ca107824eaba403e01898c21b0bR43))

When the trial has ended, it should be shown in another place. Maybe in the first field (`price_text`)?